tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/contact/openpaas/OpenPaasContactsConsumer.java

```diff
@@ -0,0 +1,80 @@
+package com.linagora.tmail.james.jmap.contact.openpaas;
+
+import jakarta.inject.Inject;
+
+import org.apache.james.backends.rabbitmq.ReactorRabbitMQChannelPool;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.linagora.tmail.james.jmap.contact.TmailContactUserAddedEvent;
+
+import java.io.IOException;
+import reactor.core.Disposable;
+import scala.NotImplementedError;
+
+
+public class OpenPaasContactsConsumer {
+    private final Logger LOGGER = LoggerFactory.getLogger(OpenPaasContactsConsumer.class);
+
+    private static final String QUEUE_NAME = "contacts:contact:add";
```

Comment:
Please let's create our own queue for TMail and not attempt to reuse the OpenPaaS one. Otherwize we are stilling messages from OpenPaaS (and it might do valuable things with it)